docs/custom-server.rst

```diff
@@ -102,3 +102,66 @@ The following example presents a basic GraphQL server using a Django framework::
         status_code = 200 if success else 400
         # Send response to client
         return JsonResponse(result, status=status_code)
+
+
+Basic GraphQL server with Flask
```

Comment:
```suggestion
Integrating with Flask
```

"Basic" could be taken to mean "that's not suitable for real-world use".

Reply:
Starting chapters with framework name will make for better copy  and navigation IMHO:

```
- Django integration
- Flask integration
- Starlette integration
```